cpp/include/cudf/utilities/traits.hpp

```diff
@@ -136,27 +136,6 @@ struct is_numeric_impl {
   }
 };
 
-/**
- * @brief Indicates whether the type `T` is a unsigned numeric type.
- *
- * @tparam T  The type to verify
- * @return true `T` is unsigned numeric
- * @return false  `T` is signed numeric
- **/
-template <typename T>
-constexpr inline bool is_unsigned()
-{
-  return std::is_unsigned<T>::value;
-}
-
-struct is_unsigned_impl {
-  template <typename T>
-  bool operator()()
-  {
-    return is_unsigned<T>();
-  }
-};
-
```

Comment:
I just moved the is_unsigned definitions from here since they were oddly stuck within the is_numeric definitions.